package.json

```diff
@@ -14,11 +14,13 @@
     "@emotion/styled": "^11.11.0",
     "@mui/icons-material": "^5.15.2",
     "@mui/material": "^5.15.2",
+    "@reduxjs/toolkit": "^2.0.1",
     "axios": "^1.6.3",
-    "bootstrap": "^5.3.2",
+    "jwt-decode": "^4.0.0",
```

Comment:
We don't need jwt-decode as the data needed from the decoded token is already coming from the backend in the login response.